setup.py

```diff
@@ -26,7 +26,8 @@ def read(*rnames):
                 break
         return line + README.read()
 
-requires=[ 'nose', 'six' ]
+
+requires = ['nose', 'six', 'mock==3.0.5']
```

Comment:
Please revert the change to the required dependencies.

Reply:
Updated